models/src_twilio.yml

```diff
@@ -283,8 +283,8 @@ sources:
       - name: message
         identifier: "{{ var('twilio_message_identifier', 'message' )}}"
         description:  Each record represents an individual outbound or inbound message.
-        meta:
-          is_enabled: "{{ var('using_twilio_message', true) }}"
+        config:
+          enabled: "{{ var('using_twilio_message', true) }}" # this var is DEPRECATED and used nowhere else
```

Comment:
Should we consider removing this altogether?

Reply:
I think that would be desirable to avoid any confusion in the future.

Reply:
removed and noted in the changelog!